pkg/apis/flowcontrol/validation/validation_test.go

```diff
@@ -1304,65 +1302,32 @@ func TestValidateLimitedPriorityLevelConfiguration(t *testing.T) {
 
 	tests := []struct {
 		requestVersion    schema.GroupVersion
-		allowZero         bool
 		concurrencyShares int32
 		errExpected       field.ErrorList
 	}{{
-		requestVersion:    flowcontrolv1beta1.SchemeGroupVersion,
-		concurrencyShares: 0,
-		errExpected:       errExpectedFn("assuredConcurrencyShares", 0, "must be positive"),
-	}, {
-		requestVersion:    flowcontrolv1beta2.SchemeGroupVersion,
-		concurrencyShares: 0,
-		errExpected:       errExpectedFn("assuredConcurrencyShares", 0, "must be positive"),
-	}, {
 		requestVersion:    flowcontrolv1beta3.SchemeGroupVersion,
```

Comment:
`v1beta3` will no longer be served starting with v1.32, do we need to keep v1beta3 validation?

Reply:
@tkashem  yes, we need it. please see https://github.com/kubernetes/kubernetes/pull/127017#issuecomment-2323744226